bundles/org.openhab.binding.evcc/README.md

```diff
@@ -53,35 +53,46 @@ Please note that some of them are only available when evcc is properly configure
+| loadpointN#chargePower              | Number:Power           | R          | Current power of charging                                                 |
+| loadpointN#chargedEnergy            | Number:Energy          | R          | Energy charged since plugged-in                                           |
+| loadpointN#charging                 | Switch                 | R          | Loadpoint is currently charging                                           |
+| loadpointN#enabled                  | Switch                 | R          | Charging enabled (mode is not "off")                                      |
+| loadpointN#maxCurrent               | Number:ElectricCurrent | RW         | Maximum amperage per connected phase with which the car should be charged |
+| loadpointN#minCurrent               | Number:ElectricCurrent | RW         | Minimum amperage per connected phase with which the car should be charged |
+| loadpointN#mode                     | String                 | RW         | Charging mode: "off", "now", "minpv", "pv"                                |
+| loadpointN#phases                   | Number                 | RW         | The maximum number of phases which can be used                            |
+| loadpointN#limitEnergy              | Number:Energy          | RW         | Amount of energy to charge the vehicle with                               |
+| loadpointN#limitSoC                 | Number:Dimensionless   | RW         | Until which state of charge (SoC) should the vehicle be charged           |
+| loadpointN#title                    | String                 | R          | Title of loadpoint                                                        |
+| loadpointN#vehicleConnected         | Switch                 | R          | Whether vehicle is connected to loadpoint                                 |
+| loadpointN#vehicleConnectedDuration | Number:Time            | R          | Duration the vehicle is connected to loadpoint                            |
+| loadpointN#vehicleCapacity          | Number:Energy          | R          | Capacity of EV battery                                                    |
+| loadpointN#vehicleOdometer          | Number:Length          | R          | Total distance travelled by EV                                            |
+| loadpointN#vehiclePresent           | Switch                 | R          | Whether evcc is able to get data from vehicle                             |
+| loadpointN#vehicleRange             | Number:Length          | R          | Battery range for EV                                                      |
+| loadpointN#vehicleSoC               | Number:Dimensionless   | R          | Current State of Charge of EV                                             |
+| loadpointN#loadpointVehicleTitle    | String                 | R          | Name of EV                                                                |
```

Comment:
```suggestion
| loadpointN#vehicleName    | String                 | R          | Name of the EV at the loadpoint                                                |
```

The second `loadpoint` in the channel name is wrong in `README.md`. Remove `loadpoint` from the channel name ...

In addition, this evcc API parameter is now called `vehicleName` instead of `vehicleTitle`.
This needs to be changed also in `Loadpoint.java`. Right now the channel is never updated as it goes for Title instead of Name.